provider/datagen/src/transform/cldr/units/prefix.rs

```diff
@@ -0,0 +1,44 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+
+struct Prefix {
+    base: u32,
+    power: i16,
+    prefix: &'static str,
+}
+
+enum SIPrefix {
+    YOTTA = Prefix { base: 10, power: 24, prefix: "yotta" },
+    ZETTA = Prefix { base: 10, power: 21, prefix: "zetta" },
+    EXA = Prefix { base: 10, power: 18, prefix: "exa" },
+    PETA = Prefix { base: 10, power: 15, prefix: "peta" },
+    TERA = Prefix { base: 10, power: 12, prefix: "tera" },
+    GIGA = Prefix { base: 10, power: 9, prefix: "giga" },
+    MEGA = Prefix { base: 10, power: 6, prefix: "mega" },
```

Comment:
Make it to methods for analyzing the string.

Reply:
`parse_prefix(s: &str) -> Option<(u8, u8)>`

Reply:
we can use the function as @robertbastian  suggest and get it to it back later

Reply:
Should be `parse_prefix(s: &str) -> Option<(u8, u8, usize)>` where the return type in `(base, power, index)`